README.md

```diff
@@ -31,3 +31,5 @@ Clicking 'Download export' will then return a CSV with columns for any `Output`
 This feature provides a task `Set Form Outputs` which will send all entries from the form to the `Output` of the workflow with the same names as their form `input`s.
 
 It has an option for `Prefix` which will prepend the entered string to the output keys.
```

Comment:
I'd probably neaten this up by saying something like.

```
The following options are available

### Prefix

Prepend the entered string....

### Ignored

Specify fields which should...
```

Thinking that if more options are added it will be clear had to add that option to the docs instead of having to write, "It also has...".